ClientUpdater/Updater.cs

```diff
@@ -851,7 +855,13 @@ private static void ExecuteScript(string fileName)
 
             try
             {
-                SafePath.DeleteFileIfExists(GamePath, key);
+                FileInfo fileInfo = SafePath.GetFile(GamePath, key);
+
+                if (fileInfo.Exists)
+                {
+                    fileInfo.IsReadOnly = false;
+                    fileInfo.Delete();
+                }
```

Comment:
Instead of constantly checking `IsReadOnly` before deleting, it is better to modify the Rampastring.Tools class `SafePath` to support disabling the read-only property in delete methods.

Reply:
I agree but let's deal with the emergency first